controllers.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
I'm not keen on defining everything inside module.exports, I feel that it detracts a little from readability. Another option is to declare each function separately as a key of module exports. eg:
`exports.getTopics = ()=>{}`